src/mower_logic/src/mower_logic/behaviors/DockingBehavior.cpp

```diff
@@ -73,8 +86,8 @@ bool DockingBehavior::approach_docking_point() {
     exePathGoal.controller = "FTCPlanner";
     ROS_INFO_STREAM("Executing Docking Approach");
 
-    auto approachResult = mbfClientExePath->sendGoalAndWait(exePathGoal);
-    if (approachResult.state_ != approachResult.SUCCEEDED) {
+    auto approachResult = waitForResultOrAborted(mbfClientExePath, exePathGoal);
+    if (aborted || approachResult != actionlib::SimpleClientGoalState::SUCCEEDED) {
```

Comment:
The `.state_` went missing in the comparison.

Reply:
Fixed.